### ISSUE TITLE
Manage Text::CSV options and methods more transparently

### ISSUE THREAD

**Issue by hertzhaft** [zoffixznet/Mojo-CSV#3]
Hi, 

I like your module very much, especially because I enjoy the "Mojo way"!

This is a feature suggestion. I have a lot of tab-separated files and would like to process them with Mojo::CSV. Currently I have to fiddle with the internal "_obj" property to set alternative options like "sep_char". That's a little cumbersome, it would be more elegant if I could set them directly through the "new" constructor (like in Text::CSV).

Similarly, it would be cool if the TexT::CSV options could also be set individually by chainable accessors (the Mojo way is not possible in Text::CSV, because the setters don't return the instance). 

Or expose even more Text::CSV methods, for example to allow error checking without having to consult _obj?

Thanks,
Martin

**Comment by zoffixznet** [comment]
To be honest, I don't program Perl 5 anymore, so it's highly unlikely I'd be adding any new features to my Perl 5 modules.

[Perl 6](https://perl6.org/) is where the party's at.

**Comment by hertzhaft** [comment]
Yeah, I know that you are committed ... :-)

Any suggestions how I should proceed? So far I've never done a pull request, would you accept one?
Or roll my own module?

**Comment by zoffixznet** [comment]
> So far I've never done a pull request, would you accept one?

Yes, as long as it doesn't break existing functionality. This module is used by one of the apps I use at work, so I wouldn't want that app to break...



> Or roll my own module?

I don't see anything wrong with that option. You could fork this module and extend it anyway you want, without a need for anyone's approval of pull requests :smile_cat:
